packages/docusaurus-theme-classic/src/theme/Tabs/index.tsx

```diff
@@ -103,7 +111,9 @@ function Tabs(props) {
       <div role="tabpanel" className="margin-vert--md">
         {
           Children.toArray(children).filter(
-            (child) => child.props.value === selectedValue,
+            (child) =>
+              (child as React.Component<{value: string}>).props.value ===
```

Comment:
```suggestion
              (child as ReactElement<{value: string}>).props.value ===
```

Reply:
the childs are elements not components (actually surprised that `Comp.props.value` works ^^)